backend/backend-cft.yaml

```diff
@@ -0,0 +1,175 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for testbed backend instance and rds connection
+Parameters:
+  HostPort:
+    Type: String
+    Default: 4500
+  ParamContainerPort:
+    Type: String
+    Default: 4500
+  ApiImage:
+    Type: String
+    Default: ga4gh/ga4gh-testbed-api:test
+  VpcId:
+    Type: String
+    Default: vpc-0d772394df78b1311
+  SubnetA:
+    Type: String
+    Default: subnet-0be7a1fb3a96191b4
+  SubnetB:
+    Type: String
+    Default: subnet-0c465ec61b98bffac
+  SubnetC:
+    Type: String
+    Default: subnet-07b0e79f4d9bbf9d0
+  DomainName:
+    Type: String
+    Default: testbed.ga4gh.org
+  DBHost:
+    Type: String
+    Default: dev-ga4gh-testbed-database-database-bqdkifegcrpe.c0q4keadjybw.us-east-2.rds.amazonaws.com
+  DBName:
+    Type: String
+    Default: testbed_db
+  DBPort:
+    Type: String
+    Default: 5432
+  DBUser:
+    Type: String
+    Default: testbed_writer
+  DBPass:
+    Type: String
+  DBSchema:
+    Type: String
+    Default: public
+  TestbedLG:
+    Type: String
+    Default: testbed-backend-lg
+  Region:
+    Type: String
+    Default: us-east-2
+  Prefix:
+    Type: String
+    Default: ecs
+  ExecutionRole:
+    Type: String
+    Default: arn:aws:iam::282236276875:role/ECSFargateExecutionRole-TT
+  SecurityGroupId:
+    Type: String
+    Default: sg-036fa481aa113d84d
```

Comment:
remove the hardcoding of security group. Use the output variable of the static template to get this value instead. You could use registry templates for reference

Reply:
Help me understand best way to handle this.
1. Option you are suggesting: Using output variable from database template same way done in registry.
2. Having it define at runtime, like it is right now - defaults are for dev instance only, during prod everything should be defined as parameters, isnt it? I am thinking of a scenario in production system where these values could be defined when running the CF template.

Reply:
I recommend going with option 1